.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   root: true,
-  extends: "babel",
+  parser: "lightscript-eslint",
```

Comment:
(re the parser, though, I can tell you that the extends: babel loads babel-eslint as the parser)